docs/snippets/ov_model_snippets.cpp

```diff
@@ -100,12 +100,10 @@ void visualize_example(const std::shared_ptr<ov::Model>& m) {
 
 void model_inputs() {
 ov::Core core;
-std::shared_ptr<ov::Model> model = core.read_model("model.xml");
+std::shared_ptr<ov::Model> ov_model = core.read_model("model.xml");
 //! [all_inputs_ouputs]
-/* Take information about all topology inputs */
-auto inputs = model->inputs();
-/* Take information about all topology outputs */
-auto outputs = model->outputs();
+auto ov_model_input = ov_model->input(index);
```

Comment:
it's better to add a separate example for indices, not to update the old one. I think we should leave the old one unchanged.

Reply:
Examples separated in https://github.com/openvinotoolkit/openvino/pull/23482/commits/a168d84821a71a806e05cdea3eb7d0101fa93d6b